### ISSUE TITLE
Adjust the size of Plot window via window edge handles

### ISSUE THREAD

**Issue by mkraete** [DC-analysis/DCscope#51]
I´m using ShapeOut2.2.4. When plotting graphs, I´d like to adjust the plot window size.

**Comment by paulmueller** [comment]
Is this the option you are looking for?

![image](https://user-images.githubusercontent.com/2853022/94522766-71734e80-0230-11eb-925d-0fb6abc972b6.png)

**Comment by mkraete** [comment]
Yes and No, all other windows can be adjusted by tracking the edge....while the plot window not.
